src/randoop/JunitFileWriter.java

```diff
@@ -153,6 +143,19 @@ private File writeSubSuite(List<ExecutableSequence> sequencesForOneFile, String
     return file;
   }
 
+/*
+ * Writes a test method to the output stream for the sequence s with name "test"+testCounter. 
+ */
+  private void writeTest(PrintStream out, String className, int testCounter, ExecutableSequence s) {
+    out.println("  @Test");
+    out.println("  public void test" + testCounter + "() throws Throwable {");
```

Comment:
Why are the created tests instance methods?  Is there a reason they cannot be static methods?  Does JUnit require use of instance methods?